app/views/spree/components/cart/_item_info.html.erb

```diff
@@ -7,13 +12,15 @@
     <%= variant.options_text %>
   </p>
 
-  <% if line_item.insufficient_stock? %>
+  <% if stock_info && line_item.insufficient_stock? %>
```

Comment:
I think this should probably be another component called line_item_availability, but we can do that in another PR. Can you please just note this somewhere so we don't keep track of it?